frontend/src/pages/index.js

```diff
@@ -60,7 +60,7 @@ class HomePage extends Component<Props> {
   }
 
   query = () => {
-    if (this.props.candidates.length == 0) {
+    if (this.props.candidates.length != 0) {
```

Comment:
It wasn't querying when we changed the filters so we changed this.

Reply:
@CodeBrew28

Reply:
I dont think you should since if you change it to != 0 that means it wont load in all the candidates. Initially the length of the candidates array is 0 so it'll fetch them all for display (at least to my understanding).